operations/research-and-development/product/release-process/release-overview.md

```diff
@@ -1,33 +1,31 @@
 # Release Overview
 
-Mattermost ships with a new version on the 16th of each month in [binary form](http://docs.mattermost.com/administration/upgrade.html#mattermost-team-edition).
+Mattermost ships with a new version on the 16th of each month in [binary form](http://docs.mattermost.com/administration/upgrade.html#mattermost-team-edition) for self-hosted customers. Mattermost also ships releases with new features and bug fixes [to Mattermost Cloud](https://docs.mattermost.com/guides/cloud-admin-guide.html) every two weeks.
```

Comment:
The two links are now broken due to the documentation re-structuring, and I'm not 100% sure where those used to link to.

Reply:
Oh great catch, thank you. I'll fix those.